aspnetcore/tutorials/publish-to-iis.md

```diff
@@ -0,0 +1,138 @@
+
+Download the installer using the following link:
+
+[Current .NET Core Hosting Bundle installer (direct download)](https://www.microsoft.com/net/permalink/dotnetcore-current-windows-runtime-bundle-installer)
+
+1. Run the installer on the IIS server.
+
+1. Restart the server or execute **net stop was /y** followed by **net start w3svc** in a command shell.
+
+## Create the IIS site
+
+1. On the IIS server, create a folder to contain the app's published folders and files. In a following step, the folder's path is provided to IIS as the physical path to the app.
+
+1. In IIS Manager, open the server's node in the **Connections** panel. Right-click the **Sites** folder. Select **Add Website** from the contextual menu.
+
+1. Provide a **Site name** and set the **Physical path** to the app's deployment folder that you created. Provide the **Binding** configuration and create the website by selecting **OK**.
+
+## Create an ASP.NET Core Razor Pages app
+
+Follow the <xref:getting-started> tutorial to create a Razor Pages app.
+
+## Publish and deploy the app
+
+*Publish an app* means to produce a compiled app that can be hosted by a server. *Deploy an app* means to move the published app to a hosting system. The publish step is handled by the [.NET Core SDK](/dotnet/core/sdk), while the deployment step can be handled by a variety of approaches. This tutorial adopts the *folder* deployment approach, where:
+
+* The app is published to a folder.
+* The folder's contents are moved to the IIS site's folder (the **Physical path** to the site in IIS Manager).
+
+# [Visual Studio](#tab/visual-studio)
+
+1. Right-click on the project in **Solution Explorer** and select **Publish**.
+1. In the **Pick a publish target** dialog, select the **Folder** publish option.
+1. Set the **Folder or File Share** path.
+   * If you created a folder for the IIS site that's available on the development machine as a network share, provide the path to the share. The current user must have write access to publish to the share.
+   * If you're unable to deploy directly to the IIS site folder on the IIS server, publish to a folder on removeable media and physically move the published app to the IIS site folder on the server, which is the site's **Physical path** in IIS Manager. Move the contents of the *bin/Release/{TARGET FRAMEWORK}/publish* folder to the IIS site folder on the server, which is the site's **Physical path** in IIS Manager.
+
+# [Visual Studio Code / .NET Core CLI](#tab/visual-studio-code+netcore-cli)
```

Comment:
I think you should just use VSC or .NET Core CLI, probably CLI
This isn't an approved tab
cc @scottaddie  @tdykstra

Reply:
Prior guidance was to combine tabs. @scottaddie said that that the error is a false positive. Currently, this tab is used in six topics.

Reply:
We've got approval to combine VSC/Mac. You can't combine any two tabs. Go with one or the other, they're both not required. I recommend CLI. Folks will figure this out.

Reply:
Sure thing ... noting that this is a change in my guidance. I specifically asked about this tab and that was the answer that I was given. I'll fix it now.